ckanext/data_qld_theme/templates/group/snippets/group_item.html

```diff
@@ -30,7 +30,7 @@
     {% endblock %}
     {% block datasets %}
       <td>
-        {% if c.controller in ['package', 'dataset'] and c.action in ['groups'] %}
+        {% if h.check_access('member_create', group) and c.controller in ['package', 'dataset'] and c.action in ['groups'] %}
```

Comment:
Shouldn't this check for `member_delete` access?

Reply:
CKANs member_delete auth function just calls member_create.
https://github.com/ckan/ckan/blob/master/ckan/logic/auth/delete.py#L139
I will update it though so it makes more sense.